### ISSUE TITLE
Can I train multi object action detection?

### ISSUE THREAD

**Issue by Batman-97** [yjh0410/YOWOv2#6]
Hello @yjh0410 ,

I have dataset with two action classes, and i need to train it, but my dataset has multi objects in same frame will the YOWO model localize two actions in single frame?

thank you in advance

**Comment by yjh0410** [comment]
@Batman-97 Certainly. For instance, the YOWOv2 trained with AVA dataset can detect multiple action instances in the same frame, as demonstrated in some qualitative results in the README file. This suggests that YOWOv2 may also be capable of handling your dataset. However, you will need to preprocess your dataset into a suitable format to train YOWOv2 using this project's code. You can refer to the format of datasets such as UCF101-24 or AVA as a guide for this. Unfortunately, I have not yet studied how to customize the project code for training on new datasets, so I am unable to provide more specific suggestions.

**Comment by Batman-97** [comment]
@yjh0410  thank you fopr your response.
my second question was about custom dataset training!!

in ucf-24 dataset there are testlist01.txt and trainlist01.txt, finalAnnots.mat what is this exactly for?? 

thank you in advance

**Comment by Batman-97** [comment]
@yjh0410  how to use pretrained yowo2 Model?? for transfer learning.

thank you

**Comment by Batman-97** [comment]
@yjh0410 你好，
我想问你我已经创建了一个自定义数据集并在其上运行训练我是成功的，但是在运行评估时即使数据集太小也需要很多时间，为什么会这样？？你有什么主意吗？？

先感谢您。

eval ...
Metric: Frame mAP
[0 / 4]
calculating Frame mAP ...
D:\Shamanth\YOWO_test\YOWOv2\evaluator\groundtruths_dataset

它永远留在这里

**Comment by zhengjiedna** [comment]
> nnots.mat这

UCF数据集在 eval 的时候有问题 groundtruths 里面的文件找不到对应的 在testlist 中找不到对应的所以 ap 永远都是0

**Comment by Eurususu** [comment]
> @yjh0410 thank you fopr your response. my second question was about custom dataset training!!
> 
> in ucf-24 dataset there are testlist01.txt and trainlist01.txt, finalAnnots.mat what is this exactly for??
> 
> thank you in advance

so have you figured out how to extract like finalAnnots.mat files?

**Comment by snehashis1997** [comment]
hi @Batman-97 
if I want to do custom training. should I mark each and every human in a frame with some class label? in my case (suppose try to build a running action detection), in every frame, there are 5-10 people. but only 1-2 are running. should I annotate others too?
